bravado/mapping/swagger20_validator.py

```diff
@@ -0,0 +1,39 @@
+from jsonschema import validators, _validators
+from jsonschema.exceptions import ValidationError
+from jsonschema.validators import Draft4Validator
+
+
+def ignore(_validator, *args):
+    """A validator which performs no validation. Used to `ignore` some schema
+    fields during validation.
+    """
+    return
+
+
+def required_validator(validator, req, instance, schema):
+    """Swagger 2.0 expects `required` to be a bool in the Parameter object,
+    but a list of properties everywhere else.
+    """
+    # TODO: Find a better way to identify if it is a Parameter schema
+    if schema.get('in'):
+        if req is True and not instance:
+            return [ValidationError("%s is required" % schema['name'])]
+    else:
+        return _validators.required_draft4(validator, req, instance, schema)
```

Comment:
The author adheres towards backwards compatibility, above is a non-public method though. Other option is to copy the [lines](https://github.com/Julian/jsonschema/blob/e1c30237cf57a0c221ea5c9a2677ca2edf20b18e/jsonschema/_validators.py#L285-290) in a local method here, which should not be an issue as there no other associations to hidden methods/attributes inside of the method.